lib/node/runner/test.go

```diff
@@ -123,7 +183,7 @@ func createNodeRunnerForTesting(n int, conf *consensus.ISAACConfiguration, recv
 	}
 
 	address := kp.Address()
-	balance := common.BaseFee.MustAdd(common.BaseReserve)
+	balance := common.BaseFee.MustAdd(1000000000000)
```

Comment:
Why?

Reply:
@Geod24 For the test, there is need to create account which have more units than BaseReserve.

Reply:
The only difference between this function and `createNodeRunnerForTesting` is the initial balance, isn't it ?
Then just make the initial balance a parameter. Another possibility is to reduce the initial balance of account, as I assume your problem is that unfreezing an account will go over the `MaximumBalance`.

Also, I asked "why" because the amount looks rather arbitrary. It does not related to `BaseReserve` in any way.
If you used something like `common.BaseReserve.MustAdd(common.BaseFee)` it would be more explicit for example.